intro-HPC/ch_MATLAB.tex

```diff
@@ -0,0 +1,117 @@
+\chapter{MATLAB}
+\label{ch:matlab}
+
+\section{Why is the MATLAB compiler required?}
+
+The main reason behind this alternative way of using MATLAB is licensing: only
+a limited number of MATLAB sessions can be active at the same time. However, once
+the MATLAB program is compiled using the MATLAB compiler, the resulting stand-alone
+executable can be run without needing to contact the license server.
+A license is required for the MATLAB Compiler, see \url{https://nl.mathworks.com/help/compiler/index.html}.
+
+\ifgent
+Only a limited amount of MATLAB sessions can be active at the same time because
+there are only a limited amount of MATLAB research licences available on
+the \university MATLAB license server. Since there are a lot of compute nodes,
+if each node would need a license, the licenses would quickly run out.
+\fi
+
+\section{How to compile MATLAB code}
+
+Compiling MATLAB code can only be done from the login nodes, because only login
+nodes can access the server, workernodes on clusters cannot.
+
+To access the MATLAB compiler, the MATLAB module should be loaded first. Make sure
+you are using the same MATLAB version to compile and to run the compiled MATLAB
+program.
+
+\begin{prompt}
+%\shellcmd{module avail MATLAB}%
+----------------------/apps/gent/CO7/sandybridge/modules/all ----------------------
+   MATLAB/2016b    MATLAB/2017b    MATLAB/2018a (D)
+%\shellcmd{module load MATLAB/2018a}
+\end{prompt}
+
+After loading the module, the \verb|mcc| command can be used. To get help on
+\verb|mcc|, you can run \verb|mcc -?|.
+
+To compile a single file, the \verb|-m|
```

Comment:
`-m` has nothing to do with a single file or not, it's for compiling "standalone applications"